packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -189,6 +192,15 @@ const PCI_ACTIONS = [
   },
 ]
 
+const SECUREBOOT_STATUS_MESSAGES = {
+  disabled: _('secureBootStatusNotEnforced'),
+  first_boot: _('secureBootStatusPending'),
+  ready: _('secureBootStatus'),
+  ready_no_dbx: _('secureBootStatusNoDbx'),
+  setup_mode: _('secureBootWantedButDisabled'),
+  certs_incomplete: _('secureBootWantedButCertificatesMissing'),
```

Comment:
I initially thought all message identifiers were to start with `secureBootStatus`.

Reply:
Some identifiers are already pretty long and I wanted to keep a certain consistency, but reading this again I see I failed at that in some of them... Maybe change this one to just ```secureBootNoDbx``` or change them all to start with ```securebootStatus```